scripts/enforce-property-single-quote-in-bundle-message

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+
+# Find the string that have `{number}`, but have only single `'` character
+non_escaped_single_quotes=$(git grep -Ph -e '=.*\{\d+\}' --and -e "(='[^'])|([^']'[^'])|([^']'$)" -- '*.properties')
+
+# Find the strings that DON'T have `{number}`, but have a double `'` character
+unnecessarily_escaped_single_quotes=$(git grep -Ph --not -e '=.*\{\d+\}' --and -e "''" -- '*.properties')
+
+
+if [[ $non_escaped_single_quotes ]]; then
```

Comment:
Swap this `if` and `unnecessarily_escaped_single_quotes=` since you don't care about the value of `unnecessarily_escaped_single_quotes` if this condition is true